packages/components/toaster/src/__tests__/Toaster.spec.tsx

```diff
@@ -0,0 +1,169 @@
+import { useState } from 'react';
+import {
+  fireEvent,
+  render,
+  screen,
+  waitFor,
+} from '../../../../../tests/test-utils';
+import Toaster from '../Toaster';
+import Button from '@douro-ui/button';
+import { ToasterStack } from '../ToasterStack';
+import { ToasterProps } from '../toaster.types';
+
+describe('<Toaster />', () => {
+  it('should renders with success type and correct position', () => {
+    render(
+      <Toaster typeToaster="success" position="bottom-center">
+        Success Message
+      </Toaster>,
+    );
+
+    expect(screen.getByText('Success Message')).toBeInTheDocument();
+    const toaster = screen.getByText('Success Message').closest('div');
```

Comment:
Setting a closest div could be dangerous. I would prefer to have a 'data-testid' to be accurate.

Reply:
Thanks for the advice.